routes/comments.js

```diff
@@ -4,7 +4,12 @@ import Comment from "../controllers/commentController";
 const comment = Router();
 
 comment.post("/articles/:slug/comments", Comment.create);
+comment.get("/articles/:slug/comments", Comment.fetchAllComments);
+comment.get("/articles/:slug/comments/:id", Comment.fetch);
```

Comment:
Can you please combine the routes in line 7, 8 , 9 into one line to avoid duplication